config/v3_0/types/storage_test.go

```diff
@@ -148,3 +148,105 @@ func TestStorageValidate(t *testing.T) {
 		}
 	}
 }
+
+func TestStorageValidateWarnings(t *testing.T) {
+	tests := []struct {
+		in  Storage
+		at  path.ContextPath
+		out error
+	}{
+		{
+			in: Storage{
+				Disks: []Disk{
+					{
+						Device: "/dev/sda",
+						Partitions: []Partition{
+							{}, {},
+						},
+					},
+				},
+				Filesystems: []Filesystem{
+					{
+						Device: "/dev/sda",
+					},
+				},
+			},
+			out: errors.ErrPartitionOverwritten,
+			at:  path.New("", "device", 0),
+		},
+		{
+			in: Storage{
+				Disks: []Disk{
+					{
+						Device:    "/dev/sda",
+						WipeTable: &[]bool{true}[0],
```

Comment:
`util.BoolToPtr(true)`

Reply:
Now thats a nice to have! thank you

Reply:
Looks like these haven't been updated.

Reply:
Yeah, idk where those changes went. I do recall doing that, but its done now.